### PR TITLE
React jwt module

### PR DESCRIPTION
**Pull request by DamnClin**
Close https://github.com/jhipster/jhipster-lite/issues/2531

The current version don't work on not styled react, hopefully this will fix that

### CODE REVIEW

**Comment by DamnClin** [comment]
Needs https://github.com/jhipster/jhipster-lite/pull/2587 for line break fix on windows

**Comment by DamnClin** [comment]
Even if we decide to disable JWT I think this should be merged (and disabled latter on)

File: src/main/java/tech/jhipster/lite/generator/project/domain/GeneratorAction.java [hunk 1]
```diff
@@ -25,7 +25,7 @@ private GeneratorAction() {}
   public static final String SPRINGBOOT_JWT = "springboot-jwt";
   public static final String SPRINGBOOT_JWT_BASIC_AUTH = "springboot-jwt-basic-auth";
 
-  public static final String REACT_JWT = "react-jwt";
+  public static final String GITHUB_ACTIONS = "github-actions";
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/generator/project/domain/GeneratorAction.java:28]: No need this, as github actions has been migrated

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

**Comment by pascalgrimaud** [comment]
can you resolve the conflict plz @DamnClin ?

**Review by pascalgrimaud — APPROVED** [review]

**Pull request merged**